packages/dds/sequence/package.json

```diff
@@ -89,10 +89,10 @@
 		"@fluidframework/build-common": "^2.0.0",
 		"@fluidframework/build-tools": "0.24.0-192663",
 		"@fluidframework/eslint-config-fluid": "^2.1.0",
-		"@fluidframework/gitresources": "^1.0.1",
+		"@fluidframework/gitresources": "^2.0.0",
 		"@fluidframework/mocha-test-setup": "workspace:~",
 		"@fluidframework/sequence-previous": "npm:@fluidframework/sequence@2.0.0-internal.6.3.0",
-		"@fluidframework/server-services-client": "^1.0.1",
+		"@fluidframework/server-services-client": "^2.0.0",
```

Comment:
Tangent to this PR: curious why a dds has (dev) dependencies on these two server bits (and no imports from them AFAICT). @Abe27342  / @anthony-murphy / @scarlettjlee do you know?

Reply:
I don't know. If I had to guess, it's probably from some really old test code (packages/dds/sequence/src/test/testFarm.ts) that has since had the bad dep removed.

Another possibility would be that someone wanted to use something that server-services-client re-exports from protocol-definitions and chose the re-export arbitrarily without taking layering into consideration.

Either way, all for removing the dep :) it shouldn't be there. If you want to keep this PR mechanical and bump it that's fine too--ping me after you complete and I'll remove it.